docs/use_cases/retrieval_augmented_generation_eval_qdrant_ragas.md

```diff
@@ -0,0 +1,611 @@
+
+This article will take you through:
+
+1) **Creating a synthetic evaluation dataset**
+   a. evaluation dataset = the basics
+   b. different ways to build one
+   c. FAQs and suggestions
+2) **Metrics**
+   a. different metrics provided by RAGAS
+   b. choosing the right metrics for your use cases
+   c. learning to decode the metric values
+3) **Using the evaluation dataset to assess given metrics**
+4) **Code walkthrough and reference notebook**
+
+
+## Synthetic evaluation dataset
+
+### The basics of evaluation datasets
+
+Golden or evaluation datasets are crucial in the assessment of information retrieval systems. Often, evaluation datasets contain triplets of queries, document ids, and relevance scores. In RAG evaluation, queries are replaced by **questions**, document id is replaced by **chunk_id, chunk text, an answer, or a ground truth**, and instead of relevance scores, we leverage **context texts** to provide an optimal evaluation record. Optionally, we could add **question complexity level** - our RAG system may perform better answering questions of a certain difficulty level.
+
+But generating numerous Question-Context-Answer samples by hand from reference data would not only be tedious but also inefficient. Furthermore, questions crafted by individuals may not achieve the complexity needed for effective evaluation, thereby diminishing the assessment's accuracy. If manually creating RAG evaluation dataset is not a feasible solution, what would be? **What's an efficient and effective way of creating our RAG evaluation dataset**?
+
+Fortunately, we can use the Hugging Face Datasets library to build our evaluation datasets efficiently and effectively. Using this library, we build two Hugging Face datasets: [Source Documentation Dataset](https://huggingface.co/datasets/atitaarora/qdrant_doc) - a dataset with all documentation text with their sources, and an [Evaluation Dataset](https://huggingface.co/datasets/atitaarora/qdrant_doc_qna) - our "golden set" or reference standard, leveraged for evaluation purposes.
+
+**Generating the questions
+To make a set of questions that are clear, concise, and related to the information in the source documents, we need to use targeted snippets - selected because they contain relevant information.
```

Comment:
any details on how you target your snippets? 
(it's not indicated in your code snippet or elsewhere)

Reply:
Does this explains - 

To create a series of clear, concise, and contextually relevant questions, we utilize focused snippets. These snippets are selected based on their ability to encapsulate or directly address specific topics or questions posed within the source documents.

Reply:
that's the selection criteria. 
I'm asking about your targeting method/tools.
is this a manual selection? did you use a specific technology to do it?

Reply:
Yes , it was a manual selection and assessment